kotlin/openID4VP/src/main/java/io/mosip/openID4VP/authorizationRequest/AuthorizationRequest.kt

```diff
@@ -40,7 +44,7 @@ class AuthorizationRequest(
                 val queryStart = decodedAuthorizationRequest.indexOf('?') + 1
                 val queryString = decodedAuthorizationRequest.substring(queryStart)
                 val encodedQuery = URLEncoder.encode(queryString, StandardCharsets.UTF_8.toString())
-                val uriString = "OPENID4VP://authorize?$encodedQuery"
+                val uriString = "?$encodedQuery"
```

Comment:
why have we removed `OPENID4VP://authorize?`?
is this part of decoded string extracted at Line# 30